ACKNOWLEDGEMENTS.md

```diff
@@ -0,0 +1,3 @@
+## Acknowledgements
+
+* Credit goes to 
```

Comment:
I think if you aren't going to give credit to anyone then you don't want this file, but I suppose it is W-I-P.